Tanks/Assets/Scripts/Shell/LavaFieldScript.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LavaFieldScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public float m_LifeTime = 3f;
+    public LayerMask m_TankMask;
+
+    void Start()
```

Comment:
-Specify "private"
-It's private by default, but it's best to explicitly write that